docs/guide.md

```
@@ -75,19 +58,19 @@ By default, authentication tokens are stored in `~/.mump2p/auth.yml`. For produc
-./mump2p --auth-path /opt/mump2p/auth/token.yml login
+mump2p --auth-path /opt/mump2p/auth/token.yml login
 
 # All subsequent commands will use the same custom path
-./mump2p --auth-path /opt/mump2p/auth/token.yml publish --topic=demo --message="Hello"
-./mump2p --auth-path /opt/mump2p/auth/token.yml logout
+mump2p --auth-path /opt/mump2p/auth/token.yml publish --topic=demo --message="Hello"
+mump2p --auth-path /opt/mump2p/auth/token.yml logout
 ```
 
 **Environment Variable Support:**
 ```sh
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)